### PR TITLE
Step 1, build 3.7.2 bazel toolchains

### PR DESCRIPTION
**Pull request by wrowe**
Build with envoy toolchain for bazel 3.7.2.

- Follow this to adopt bazel 3.7.2 for envoy upon its release
- then we will bump /bazelbuild/envoy/WORKSPACE to pick up bazel 3.7.2 and validate
- then we wait hours for all builds in flight before proceeding to add llvm 11.0.0 in Windows to a rebuild of the 3.7.2 envoy build tools alone

Co-authored-by: Sunjay Bhatia <sunjayb@vmware.com>

### CODE REVIEW

**Comment by wrowe** [comment]
This PR is staged as a draft, awaiting the announcement of bazel 3.7.1 perhaps Tuesday.

File: toolchains/regenerate.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ esac
 
 # Bazel query is the right command so bazel won't fail itself.
 # Keep bazel versions here at most two: current master version, next version
-for BAZEL_VERSION in "3.6.0" "3.7.0"; do
+for BAZEL_VERSION in "3.7.0" "3.7.1"; do
```
Comment (lizan) [inline toolchains/regenerate.sh:41]: let's keep the first one 3.6.0 since 3.7.0 is not used in envoyproxy/envoy

**Comment by sunjayBhatia** [comment]
Blocked on https://github.com/bazelbuild/bazel/issues/12545 as well

**Comment by wrowe** [comment]
> Blocked on [bazelbuild/bazel#12545](https://github.com/bazelbuild/bazel/issues/12545) as well

This fix is picked up in Bazel 3.7.2rc1 which we expect be released, this week. Updating migration patches now for this switch, and testing locally against 3.7.2 candidate.

**Comment by sunjayBhatia** [comment]
@wrowe I don't think the `/retest` command and repokitteh are set up for this repo, probably have to just push an empty commit

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**